frontend/package.json

```diff
@@ -53,7 +54,7 @@
     "start:proxy-and-server-inspect": "./scripts/start-proxy-and-server.sh --inspect",
     "start": "EXTEND_ESLINT=true react-scripts start",
     "sync-backend-sample-config": "node scripts/sync-backend-sample-config.js",
-    "test": "LC_ALL=\"en_US.UTF-8\" react-scripts test",
+    "test": "LC_ALL=\"en_US.UTF-8\" react-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
For my education, why is this required?

Reply:
I have encountered the issue listed in https://github.com/testing-library/dom-testing-library/issues/477 when using `waitFor` in `MetricsTab.test.tsx`. I also have a comment there to explain this issue. The final fix should be upgrading `react-scripts` to next major version, this flag here is only temporary.